ooniprobe/View/TestResults/Footer/UploadFooterViewController.m

```diff
@@ -118,6 +119,20 @@ -(void)uploadSingleMeasurement:(Measurement*)measurement{
         [measurement setReport_id:[results updatedReportID]];
         [measurement save];
     }
+    return [results good];
 }
 
+-(void)showRetryPopup:(NSArray *)notUploaded :(int)start{
```

Comment:
Here it would be better to remove the space between `notUploaded` and `:`

Reply:
It's obj-c syntax. I can't remove the space, I can only add a name for the param, will do that.

Reply:
ah, wow, I completely missed this aspect. Yes, please!